QUEST.tsv

```diff
@@ -1407,96 +1408,96 @@ QUEST_20150317_001407	Sincerity of Moze
-QUEST_20150323_001453	If you ever see the Devout Tree, please bring it back to life. {nl}I heard that tree blocks the power of the curse.
+QUEST_20150323_001446	Those brothers are crazy. They must be completely nuts. {nl}When did it start? When Dora attacked Gracius? {nl}When they saw the demons marching?
+QUEST_20150323_001447	It became worse since Gracius was left behind injured. {nl}I kept my mouth shut while everyone's eyes turned murderous. {nl}The town was like a volcano waiting to erupt. {nl}I thought it was a lesson until then.
+QUEST_20150323_001448	Then the trigger was released. {nl}A brother I can't remember had smacked down Dion with a mace when he was sleeping and I… {nl}Everyone… {nl}No one could see their old selves desiring to become a priest. 
+QUEST_20150323_001449	This surely is not a teaching of Sir Maven. {nl}Greed? Laziness? How can call killing each other teach a lesson? {nl}I even envy Dion for dying first.
+QUEST_20150323_001450	I can hear snickering somewhere. {nl}It could be the sound of Dora deceiving me. {nl}It doesn't really matter any more.
+QUEST_20150323_001451	The trapped soul gave the scripture and disappeared.
+QUEST_20150323_001452	Witas' curse stopped working thanks to the medicine.
+QUEST_20150323_001453	If you ever see the Devout Tree, please bring it back to life. {nl}I've heard that tree can block the power of the curse.
 QUEST_20150323_001454	I believe that the evil will eventually fall and the Goddess will be victorious. {nl}The mark of victory is right in front of me.
-QUEST_20150323_001455	Anyway, I will go to the battlefield first so please help the guards and my followers. {nl}I will meet you in Nefritas Cliff.
-QUEST_20150323_001456	Do not think this is over. {nl}I will be watching from Kvailas Forest. 
+QUEST_20150323_001455	Anyway, I will go to the battlefield first, so please help the guards and my followers. {nl}I will see you on Nefrito Cliff.
+QUEST_20150323_001456	This isn't over! {nl}I'll be watching from Kvailio Forest. 
 QUEST_20150323_001457	Catacomb 1st Floor internal warp
 QUEST_20150323_001458	It needs to be purified more…
-QUEST_20150323_001459	Blurry spirit of Catacomb 1st Floor
+QUEST_20150323_001459	Blurry Spirit of Catacomb 1st Floor
 QUEST_20150323_001460	A blurry existence can be felt around the area…
-QUEST_20150323_001461	Unseen things are starting to be seen…
-QUEST_20150323_001462	Letter someone left behind
-QUEST_20150323_001463	Tree Root Crytals of Pilgrim Path is cursed by Nakits. {nl}As if to mock sir Mayvern's teachings about being careful of laziness. 
-QUEST_20150323_001464	It should be destroyed before the curse spreads. {nl}But I just can't handle the Tree Root Crystals by myself. 
-QUEST_20150323_001465	Soul of Pilgrims swallowed by the monsters can destroy the Tree Root Crystal. {nl}Maybe I too will be one of them soon.
+QUEST_20150323_001461	Beings that were once invisible are starting to appear…
```

Comment:
I think this line means that things that were never seen before are now being seen, in other words, they did not already exist as invisible beings and are now visible, but were simply never seen before ( either they did not exist, or were so foreign that nobody witnessed them ).